pkg/ruler/ruler.proto

```diff
@@ -26,6 +26,7 @@ message RulesRequest {
   string type = 4;
   string state = 5;
   string health = 6;
+  repeated string matches = 7;
```

Comment:
Let's rename it to `matchers`? `matches` sounds weird.

Reply:
+1